masschallenge/semantic-ui-theme
# AC-7915 P2: Overwrite how placeholder text should appear

**Changes introduced in [AC-7915](https://masschallenge.atlassian.net/browse/AC-7915)**
- Placeholder text looks like the mockup (color & copy)

**How to test**
- Checkout this branch
- Masquerade as a user that has access to office hours 
- Open the create modal and see that the text area has the right color and copy

[AC-7915]: https://masschallenge.atlassian.net/browse/AC-7915

[Sibling PR](https://github.com/masschallenge/accelerate/pull/2735)
